torch_xla/csrc/runtime/runtime.cc

```diff
@@ -34,13 +35,12 @@ ComputationClient* CreateClient() {
 }  // namespace
 
 ComputationClient* GetComputationClient() {
-  std::call_once(g_computation_client_once,
-                 [&]() { g_computation_client = std::move(CreateClient()); });
-  return g_computation_client.load();
+  static auto client = std::unique_ptr<ComputationClient>(CreateClient());
```

Comment:
If the static variable is guaranteed to be initialized once, do you still need the function `GetComputationClientIfInitialized` and the variable `std::atomic<bool> g_computation_client_initialized(false);`?

Reply:
The client is guaranteed to be initialized once on the first time we call into `GetComputationClient`. There are a couple of calls that want to tell if `client` has been initialized (ie comparing `GetComputationClientIfInitialized` to `nullptr`), but there's not a way I can see to tell if `client` has been initialized. As soon as you call `GetComputationClient`, it gets created. That's why I added the extra flag outside of `GetComputationClient`.

On a second look, I think it would actually be clearer to eliminate `ComputationClient* GetComputationClientIfInitialized` and replace it with `bool ComputationClientIsInitialzed()`.